instructions.md

```diff
@@ -83,13 +83,13 @@ This file describes your product. It is YAML formatted with the following root-l
 
 | Field               | Description |
 | ------------------- | ----------- |
-| `vendor`            | Name of the legal entity that is certifying. This entity must have a signed participation form on file with the CNCF  |
+| `vendor`            | Name of the legal entity that is certifying. This entity must have a signed participation form on file with the LFN  |
 | `name`              | Name of the product being certified. |
 | `version`           | The version of the product being certified. |
 | `website_url`       | URL to the product information website |
 | `repo_url`          | If your product is open source, this field is necessary to point to the primary GitHub repo containing the source. It's OK if this is a mirror. OPTIONAL  |
 | `documentation_url` | URL to the product documentation. OPTIONAL |
-| `product_logo_url`  | URL to the product's logo, (must be in SVG format -- not a PNG -- and include the product name). If not supplied, we'll use your company logo. Please see logo [guidelines](https://github.com/cncf/landscape#logos). |
+| `product_logo_url`  | URL to the product's logo, (must be in SVG format -- not a PNG -- and include the product name). If not supplied, we'll use your company logo. (TODO: LFN logo guidelines not found) Please see logo [guidelines](https://github.com/cncf/landscape#logos). |
```

Comment:
@martin-mat @lixuna @Smitholi67 Maybe these guidelines can be copied over to the [cnti certification repo](https://github.com/lfn-cnti/certification) or the [cnti main repo](https://github.com/lfn-cnti/cnti).  Either way that can be a new pull request